README.md

```diff
@@ -143,14 +143,15 @@ A *CounterEvent* envelope is dervied from a v2 *Counter* envelope
 
 #### ValueMetric
 
-A *ValueMetric* envelope is dervied from a v2 *Gauge* envelope. Each gauge
-value in a *Gauge* envelope will become a single *ValueMetric* envelope.
-
-| v1    | v2                             |
-|-------|--------------------------------|
-| name  | first-key                      |
-| value | gauge.metrics[first-key].value |
-| unit  | gauge.metrics[first-key].unit  |
+One or more *ValueMetric* envelope are dervied from a v2 *Gauge* envelope. Each
```

Comment:
can we change this:

One or more *ValueMetric* envelope

to

One or more `ValueMetric` envelopes

Reply:
Fixed.